### PR TITLE
Running a Non-Sequencing Hemi Node with Docker Compose

### PR DESCRIPTION
**Pull request by ClaytonNorthey92**
**Summary**
Added a readme + config files to run a hemi node via `docker compose`.  Any keys/secrets/credentials are meant for local communication and should be changed when/if ports are exposed to the greater internet.

**Changes**
See summary.

### CODE REVIEW

**Comment by ClaytonNorthey92** [comment]
this PR is not ready for review; I just opened it so @jcvernaleo and I (and others) can discuss and comment on code

File: localnode/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    image: alpine:latest
+    command:
+      - chown
+      - -R
+      - "65532" # geth user
+      - /tbcdata
+    volumes:
+      - tbc_data:/tbcdata
+
+  op-geth-l2:
```
Comment (ClaytonNorthey92) [inline localnode/docker-compose.yml:185]: annoyingly, it seems like we will have to p2p op-geth-l2 as well, let's add the toml file to this PR tomorrow

File: localnode/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+      BSS_BFG_URL: "ws://bfgd:8080/v1/ws/private"
+      BSS_LOG_LEVEL: "INFO"
+      BSS_ADDRESS: ":8081"
+    networks:
+      localnode:
+
+  prysm:
+    image: gcr.io/prysmaticlabs/prysm/beacon-chain:stable
+    command:
+      - "--execution-endpoint=http://geth-l1:8551"
```
Comment (ClaytonNorthey92) [inline localnode/docker-compose.yml:122]: doing some testing, let's make `geth-l1` a dependency of `prysm`

File: localnode/.env
Comment (ClaytonNorthey92) [inline localnode/.env:1]: I think we can delete this file

Comment (jcvernaleo) [reply localnode/.env:1]: removed.

**Review by joshuasing — APPROVED** [review]

**Review by jcvernaleo — APPROVED** [review]
Changes from the stuff I did look good.

OK

**Pull request merged**